vllm/v1/spec_decode/metrics.py

```diff
@@ -88,11 +97,15 @@ def log(self, log_fn=logger.info):
             "SpecDecoding metrics: "
             "Draft acceptance rate: %.1f%%, "
```

Comment:
While we are revisiting these metrics, could we move this to the end of the line and rename it to "Average (or Mean) draft acceptance rate"? I often feel like average-per-position-acceptance-rate is not a very useful metric since it varies heavily depending on the number of speculated tokens. I think that if we rename it and move next to the per-position scores, then have AL as the primary metric it would be much more readable.